governance.md

```diff
@@ -0,0 +1,164 @@
+
+There are two types of consensus models we use, the lightweight and the full:
+
+* **Lightweight consensus** occurs when a decision does not have wide or sweeping effects and is considered approved if no one raises any blocking objections after several days.
+In practice, an email is sent to a SIG or main community list, and if no one raises an objection after three days, the proposal passes.
+* **Full consensus** decisions require 3 yes votes (+1) and no objections (-1’s) and votes should be left open for at least 72 hours.
+All -1s votes require reason/ explanation, -1’s with no substantiation may be overturned.
+Thus a single -1 vote can block achieving consensus, so it forces the group to take care of the blocking person rather than outvoting.
+
+SIGs decide by consensus within themselves whatever they are doing, and which model is used for a particular decisions.
+
+Anything needing wider discussion and approval goes to mailing list for either group or SIG roll-up consensus.
+
+When we are ready, we use this power to form a technical and/or overall steering committee, with corresponding election system.
+
+#### Consensus decision making
+
+One practice of meritocracy is the consensus-based decision model.
+From http://en.wikipedia.org/wiki/Consensus_decision-making, “Consensus decision-making is a group decision making process that seeks the consent of all participants.”
+In practice, it is different from a majority-vote-wins approach.
+In the Op1st project a discussion toward a decision follows this process:
+
+1. A proposal is put forth and a check for consensus is made.
+   1. Consensus is signified through a +1 vote.
+1. A check is made for any dissent on the proposal.
+   1. Reservations? State reservation, sometimes with a ‘-1’ signifier
+      1. Reservations about the proposal are worked through, seeking consensus to resolve the reservations.
+      2. A reservation is not a vote against the proposal, but may turn into a vote against if unresolved.
+      It is often expressed with an initial -1 vote to indicate reservations and concerns.
+      This indicates there is still discussion to be had.
+   1. Stand aside?
+      No comment, or state concerns without a -1 reservation; sometimes the ‘-0’ signifier is used.
+      1. This option allows a member to have issues with the proposal without choosing to block the proposal, by instead standing aside with a +/-0 vote.
+      1. The stated concerns may influence other people to have or release reservations.
+   1. Block?
+   Vote ‘-1’ with reasons for the block.
+     1. This is a complete block on a proposal, refusing to let it pass.
+     A block is a -1 vote and must be accompanied with substantive arguments that are rooted in the merit criteria of the Project – protecting the community, the upstream, technical reasons, and so forth.
+
+
+### Use of Architectural Decision Records (ADR)
```

Comment:
not sure if we need this in the governance doc? It's important, but it's also just a technical implementation of tracking decisions.

Reply:
Maybe not in the governance doc, but (as link/doc) good as a guideline/hands on.

Reply:
I'd like to continue this discussion a bit without blocking the Governance 1.0 acceptance. I think it's OK to go ahead with a "maybe this doesn't fit" for the Gov 1.0 release, and we can change or remove for Gov 1.1 if needed.

There are a few reasons I see ADRs as fitting in the governance:

1. Governance documents are largely about how decisions are made. _If_ an ADR is only about a specific technical implementation detail, then it is probably enough to say, "SIGs capture technical decisions in ADRs, here's a link about that. A SIG may adopt additional processes around recording decisions." That is, the governance basically leaves the details of the ADRs up to the SIGs. Perhaps one SIG ends up owning the ADR process entirely, and all of the governance around that would be within that SIG.
2. This may be my own fancy, but I've been considering what it would look like to adopt ADRs for aspects of a community architecture. That is, the possibility that an Operate First environment wants to capture decisions about how and why a specific community is architected in support of a project's mission.
3. Since an inherent effect of realizing the Operate First concept as an actual running cloud environment is creating all the code and content needed to create a functional operate first environment, it may be important to specify this kind of granular detail in the governance or the Operate First model overall in some way. That is, more so than other open source projects, it is crucial for Op1st (and all Operate First environments by extension) to have a clear and fair way to track decisions. To not do so is to fail at a core technique of operating first within the open source ecosystem.

Reply:
+1 to the "without blocking" | it is OK to go ahead with a "maybe"
still need to think about @quaid 3 reasons, but they make sense to me so far

Reply:
I'm resolving this but we will definitely come back to this PR and gather all the open points for seeding the Gov 1.1 discussions.